app/admin/checkin_count.rb

```diff
@@ -0,0 +1,11 @@
+ActiveAdmin.register_page "Checkin Count" do
```

Comment:
Looks like checkin_count was already in dev, is an older version though, not your most recent.